packages/mesos-modules/buildinfo.json

```diff
@@ -3,7 +3,7 @@
     "single_source" : {
       "kind": "git",
       "git": "https://github.com/dcos/dcos-mesos-modules.git",
-      "ref": "7ce5ba397bbaa8fdcc985ef2e6dd8bc81709931a",
-      "ref_origin": "1.9.x"
+      "ref": "2a5a02f1de47dcbe44844f7358adf85ea630cf4a",
```

Comment:
Yes, that's recent enough to correspond with the next modules commit.

Basically, the makefile change corresponds to the addition of this file in Mesos:
https://github.com/mesosphere/mesos/blob/dcos-mesos-master-4faca73/3rdparty/googletest-release-1.8.0.tar.gz